extension/src/experiments/model/filterBy/quickPick.ts

```diff
@@ -67,11 +69,43 @@ export const operators = [
     label: Operator.NOT_CONTAINS,
     types: ['string'],
     value: Operator.NOT_CONTAINS
+  },
+  {
+    description: 'After Date',
+    label: Operator.AFTER_DATE,
+    types: [ColumnType.TIMESTAMP],
+    value: Operator.AFTER_DATE
+  },
+  {
+    description: 'Before Date',
+    label: Operator.BEFORE_DATE,
+    types: [ColumnType.TIMESTAMP],
+    value: Operator.BEFORE_DATE
+  },
+  {
+    description: 'On Day',
```

Comment:
Maybe it's an issue on my end but the on day filter doesn't work for me:


https://user-images.githubusercontent.com/43496356/187434628-24eb97d0-a885-4eeb-b0fd-79c98b4d1de1.mov

Reply:
It won't remove the current branch row.

Reply:
> It won't remove the current branch row.

Yes, but shouldn't experiment `exp-f659a` stay because it was created on August 30th? 

<img width="665" alt="image" src="https://user-images.githubusercontent.com/43496356/187469826-686b613d-d2e7-4a0e-bae6-b176d7031b5a.png">